to_be_ccppized/ccpp_const_utils.F90

```diff
@@ -13,13 +13,13 @@ subroutine ccpp_const_get_idx(constituent_props, name, cindex, errmsg, errflg)
     use ccpp_constituent_prop_mod, only: ccpp_constituent_prop_ptr_t
 
     ! Input arguments
-    type(ccpp_constituent_prop_ptr_t), pointer, intent(in)  :: constituent_props(:)
-    character(len=*),                           intent(in)  :: name   ! constituent name
+    type(ccpp_constituent_prop_ptr_t), intent(in)  :: constituent_props(:)
```

Comment:
Could I ask if there's any particular reason the `pointer` attribute was removed from here?

`ccpp_const_get_idx` is used in `cam_constituents` as well so this change would affect existing code - could you confirm that CAM-SIMA still passes tests after removing the `pointer` attribute here? I think it is _fine_ if the actual argument is a `pointer` (in `cam_constituents::const_props`) but the dummy argument is not, as long as it's read only, but it would be good to check...

Reply:
Thanks for pointing this out, I had meant to add a comment about this, but I forgot when I sent submitted the PR.

I removed the `pointer` attribute because we don't include this attribute for `constituent_props` in the musica module functions, and it isn't actually needed in the `ccpp_const_get_idx` function, as you point out.

However, if this function is being used through an interface (e.g., for a function pointer or an implementation of a deferred type-bound procedure) where the `pointer` attribute couldn't be removed for some reason, we could do a separate implementation of the `ccpp_const_get_idx` function for non-pointers. Given the intent of this function, it seemed pretty unlikely to me that this was the case, but I agree it's good to check. Maybe @peverwhee could weigh in?

Reply:
Thanks @mattldawson! If it isn't actually needed (should be fine if it's `intent(in)`) I'm happy to accept the change, just wanted to make sure it wouldn't break anything else. I'll leave this conversation open in case @peverwhee would like to add comments but I'll approve the PR now - no need to wait for me!

Reply:
i'm ok with removing the pointer attribute as well - I just ran the regression tests with it removed and they work fine so I'll agree with @jimmielin that it isn't needed.

Reply:
awesome, thanks!